ames_qsar/gethlenergy.py

```diff
@@ -13,9 +13,8 @@ def gethlenergy(dir_path, filename):
     b = 0
     #initialize a holding variable
     prevLine = []
-    os.chdir(dir_path)
 
-    with open(filename) as searchfile:
+    with open((dir_path + '/' + filename)) as searchfile:

```

Comment:
os.path.join(dir_path, filename)

Reply:
don't do this here, but you should know about this: "%s/%s" % (dir_path, filename)